suse_cloud_image_name_parser/suse_cloud_image_name.py

```diff
@@ -635,3 +639,13 @@ def oval_product(self):
         raise UndefinedOvalProductError(
             f'Unable to find matching oval_product for {self._image_name}'
         )
+
+    @property
+    def is_containerized(self):
+        """
+        Boolean flag that indicates if the image includes software as
+        containers or not
+        """
+        if self.is_suma and self.product_version == "5.1":
```

Comment:
Needs to be a float comparison and start with 5.0, i.e. `>= 5.0`. Multi Linux Manager has a containerized implementation starting with 5.0